civil_society_vote/hub/forms.py

```diff
@@ -72,16 +81,18 @@ def __init__(self, *args, **kwargs):
             except (ValueError, TypeError):
                 pass  # invalid input, fallback to empty queryset
 
+        self.fields["accept_terms_and_conditions"].label = mark_safe(
+            _(
+                'I agree to the <a href="https://votong.ro/ro/termeni/" target="_blank">Terms and Conditions</a> '
+                "of the VotONG platform"
+            )
+        )
+
     def clean_accept_terms_and_conditions(self):
         if not self.cleaned_data.get("accept_terms_and_conditions"):
             raise ValidationError(_("You need to accept terms and conditions."))
         return self.cleaned_data.get("accept_terms_and_conditions")
 
-    def clean_politic_members(self):
-        if not self.cleaned_data.get("politic_members"):
-            raise ValidationError(_("Organisation members need to be apolitical."))
-        return self.cleaned_data.get("politic_members")
```

Comment:
This validation should happen on `update` not on `registration` as the field is excluded during `registration`